### DISCUSSION TITLE
.ext file is optional instead of required

### DISCUSSION THREAD

**Discussion by stevechoy** [Andy00000000000/nonmem2mrgsolve#33] [Ideas]
It would be great if the .ext file is treated as optional and not required for nonmem2mrgsolve::nonmem2mrgsolve to work.

**Comment by Andy00000000000** [comment]
Hi Steve,

Agree, at some point I'm hoping to get it to a point where only the .ctl (or .mod) file is needed, with the translation setting the mrgsolve thetas/omegas to the initial NONMEM estimates. For now the .ext (or .cnv) was easy to pull from to set to final estimates, but is limited by requiring a completed NONMEM run. My workaround in the meantime has been to run MAXEVAL=1 (for FOCEI; and I believe I have MAXEVAL=0 working as well, but don't remember off top of my head) to get the .ext file without waiting too long for a complete NONMEM estimation.

Thanks for the feedback, always appreciated!

Andy

**Comment by SepassianAcademy** [comment]
Dear Andy, I have started exploring nonmemtomrgsolve. I get the correct output in the console, but for some reason I do not get the R file. I only get an empty R file. I have so far tried the 1 compartment model from the example that you have provided. What can I do in this regard? thanks in advance Ali

**Reply by Andy00000000000** [reply]
Hi Ali,

Are you getting any warning or error messages? Can you try writing a test file using the below code and let me know whether that works?

`
tmp_dir <- "/home/epd/Documents/" # change this line to your file path for writing the R file

tmp_pth <- paste0(tmp_dir,"Test.R")
tmp_file <- file(tmp_pth)
cat("'",file=tmp_pth)
cat("Hello World",file=tmp_pth,sep="\n\n",append=T)
cat("This is Working",file=tmp_pth,append=T)
cat("'",file=tmp_pth,append=T)
close(tmp_file)
`

**Reply by Andy00000000000** [reply]
Hi Ali,

I am moving this to the issues tab (https://github.com/Andy00000000000/nonmem2mrgsolve/issues/37#issue-2659475848) and will copy your follow-up into there and then delete from this discussion thread. Let's follow up in issues, thanks!
